README.md

```diff
@@ -222,3 +221,12 @@ function to use.  The following are the top level keys in the object.
 * `id` - A UUID to uniquely identify this Add-On run.  It allows DocumentCloud
   to identify the run, as well as allowing the run to send back progress,
   status message and file updates.
+
```

Comment:
We should centralize the documentation so it doesn't get fragmented and out of date - can you replace the example read me with more of a description of how this add-on in particular works?  I think we will move the documentation somewhere else, and have all the add-ons link to it.

Reply:
Good point! Having this show up in search results for general add-on setup would confuse things.

Reply:
Addressed in 3efa8f8.